src/System.Diagnostics.Debug.SymbolReader/src/System.Diagnostics.Debug.SymbolReader.csproj

```diff
@@ -7,42 +7,10 @@
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netstandard1.3</PackageTargetFramework>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
-    <!-- Disable 1685 (aka multiple type definitions) warning so it doesn't turn into an error -->
-    <NoWarn>$(NoWarn);1685</NoWarn>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the options -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
-  <!-- Work around a bug that will be fixed by buildtools/#731 -->  
-  <PropertyGroup Condition="'$(TargetGroup)' == ''">  
-    <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>  
-  </PropertyGroup>  
-  <ItemGroup Condition="'$(TargetGroup)'!='netcore50aot'">
-    <TargetingPackReference Include="mscorlib" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netcore50aot'">
-    <TargetingPackReference Include="System.Private.CoreLib" />
-    <TargetingPackReference Include="System.Private.Reflection" />
-    <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="System/Diagnostics/Debug/SymbolReader/SymbolReader.cs" />
```

Comment:
Perhaps I'm not reading the diff correctly but it looks like you removed the actual source code for this project. I don't see any .cs files listed.